Scripts/utils/validate_network.py

```diff
@@ -42,6 +42,13 @@ def validate(network, fares=None):
                 found_zone_share)
             log.error(msg)
             raise ValueError(msg)
+    validate_mode(network, param.main_mode, "AUTO")
```

Comment:
Are "AUTO", "AUX_AUTO", "TRANSIT", and "AUX_TRANSIT" here names that come from how Emme API reads networks? Consider moving these magic variables as a parameter.

Reply:
Yes, they come from the Emme text file import syntax. Now I moved the strings to the beginning of the file as constants.